tests/meta/store/test_cg_meta_store_base.py

```diff
@@ -0,0 +1,103 @@
+
+    mock_case.links[0].sample.data_analysis = None
+
+    with pytest.raises(PipelineUnknownError) as exc_info:
+        store_base.add_new_analysis(mock_bundle, mock_case, mock_status, mock_version)
+
+    assert exc_info.value.message == f"No pipeline specified in {mock_case}"
+
+
+@mock.patch("cg.store.Store")
+@mock.patch("housekeeper.store.models")
+def test_add_new_analysis_duplicate_analysis_exception(mock_housekeeper_store, mock_status):
+    """
+        test catching the exception when no pipeline is found
+    """
+    mock_bundle = mock_housekeeper_store.Bundle.return_value
+    mock_version = mock_housekeeper_store.Version.return_value
+    mock_case = mock_status.Family.return_value
+    mock_status.analysis = mock_status.analysis.return_value
+
+    with pytest.raises(AnalysisDuplicationError) as exc_info:
+        store_base.add_new_analysis(mock_bundle, mock_case, mock_status, mock_version)
+
+    assert (
+        exc_info.value.message
+        == f"Analysis object already exists for {mock_case.internal_id} {mock_version.created_at}"
+    )
+
+
+@mock.patch("cg.store.Store")
+@mock.patch("housekeeper.store.models")
+def test_add_new_analysis(mock_housekeeper_store, mock_status):
+    """
+        test adding a new analyis to cg store
+    """
+    mock_bundle = mock_housekeeper_store.Bundle.return_value
+    mock_version = mock_housekeeper_store.Version.return_value
+    mock_case = mock_status.Family.return_value
+    mock_status.analysis.return_value = None
+
+    new_analysis = store_base.add_new_analysis(mock_bundle, mock_case, mock_status, mock_version)
+
+    assert new_analysis.family == mock_case
+
+
+# @mock.patch("cg.meta.store.base._determine_missing_files")
+# def test_build_bundle(
+#     mock_missing,
+#     snapshot: Snapshot,
+#     config_data: dict,
+#     sampleinfo_data: dict,
+#     deliverables_raw: dict,
+# ):
+#     """
+#         tests the function_build bundle against a snapshot
+#     """
+#     # GIVEN the MIP analysis config data, the sampleinfo data and the deliverables file
+
+#     # WHEN building the bundle
+#     mock_missing.return_value = False, []
+#     mip_rna_bundle = store_base.build_bundle(config_data, sampleinfo_data, deliverables_raw)
+
+#     # THEN the result should contain the data to be stored in Housekeeper
+#     snapshot.assert_match(mip_rna_bundle)
+
+
+# @mock.patch("cg.meta.store.base._determine_missing_files")
+# def test_get_files(mock_missing, snapshot: Snapshot, deliverables_raw: dict):
+#     """
+#         tests the function get_files against a snapshot
+#     """
+#     # GIVEN the MIP RNA analysis deliverables file
+#     mock_missing.return_value = False, []
+#     pipeline = "wts"
+
+#     # WHEN getting the files used to build the bundle
+#     mip_rna_files = store_base.get_files(deliverables_raw, pipeline)
+
+#     # THEN the result should contain the data to be stored in Housekeeper
+#     snapshot.assert_match(mip_rna_files)
```

Comment:
Remove commented code or include it again

Reply:
fixed in https://github.com/Clinical-Genomics/cg/pull/621/commits/f4d1932a966fae2eaf831da9617630549683b588